database.py

```diff
@@ -0,0 +1,18 @@
+from flask import g
+import os
+import sqlite3
+
+DATABASE = os.path.join(os.path.abspath(os.path.dirname(__file__)), "app.db")
+
+def get_db():
+    if "db" not in g:
+        g.db = sqlite3.connect(DATABASE,
+            detect_types=sqlite3.PARSE_DECLTYPES
+        )
+        g.db.row_factory = sqlite3.Row
+    return g.db
+
+def close_db(e=None):
+    db = g.pop("db", None)
+    if db is not None:
+        db.close()
```

Comment:
Don't think we need this file anymore? We're gonna use pymysql

Reply:
I've removed that file and where it's used in app.py